firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -66,10 +66,12 @@ static boolean isValidApiKeyFormat(@Nullable String apiKey) {
     return API_KEY_FORMAT.matcher(apiKey).matches();
   }
 
-  /* Returns a random number of milliseconds less than or equal to 1000. This helps to avoid cases
-   where many clients get synchronized by some situation and all retry at once, sending requests
-   in synchronized waves. The value of random_number_milliseconds is recalculated after each retry
-  request.*/
+  /**
+   * Returns a random number of milliseconds less than or equal to 1000. This helps to avoid cases
+   * where many clients get synchronized by some situation and all retry at once, sending requests
+   * in synchronized waves. The value of random_number_milliseconds is recalculated after each retry
+   * request.
+   */
   public long getRandomMillis() {
```

Comment:
How about renaming this to one of
* `#getRandomTimeInterval`
* `#getRandomDelay`
* `#getRandomDelayForSyncPrevention`

The goal is to have a name that semantically describes what a method is used for, rather than a name that describes its implementation.